### PR TITLE
Fix: unpin flake8 and fix cs errors

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Very nice!

File: examples/distillation/scripts/extract_distilbert.py [hunk 1]
```diff
@@ -78,12 +78,12 @@
...
 
-    compressed_sd[f"vocab_projector.weight"] = state_dict[f"cls.predictions.decoder.weight"]
-    compressed_sd[f"vocab_projector.bias"] = state_dict[f"cls.predictions.bias"]
+    compressed_sd["vocab_projector.weight"] = state_dict["cls.predictions.decoder.weight"]
+    compressed_sd["vocab_projector.bias"] = state_dict["cls.predictions.bias"]
     if args.vocab_transform:
         for w in ["weight", "bias"]:
-            compressed_sd[f"vocab_transform.{w}"] = state_dict[f"cls.predictions.transform.dense.{w}"]
-            compressed_sd[f"vocab_layer_norm.{w}"] = state_dict[f"cls.predictions.transform.LayerNorm.{w}"]
+            compressed_sd[f"vocab_transform.{w}"] = state_dict["cls.predictions.transform.dense.{w}"]
```
Comment (girishponkiya) [inline examples/distillation/scripts/extract_distilbert.py:85]: I think `state_dict["cls.predictions.transform.dense.{w}"]` shoud be `state_dict["cls.predictions.transform.dense.{w}"]` (the key should be f-string).

File: examples/distillation/scripts/extract_distilbert.py [hunk 2]
```diff
@@ -78,12 +78,12 @@
...
+            compressed_sd[f"vocab_layer_norm.{w}"] = state_dict["cls.predictions.transform.LayerNorm.{w}"]
```
Comment (girishponkiya) [inline examples/distillation/scripts/extract_distilbert.py:86]: same here. The key should be f-string.

File: examples/distillation/scripts/extract_distilbert.py (hunk 1 above)
Comment (julien-c) [reply examples/distillation/scripts/extract_distilbert.py:85]: Correct – I'll fix it on master.

**Pull request merged**